pkg/reconciliation/reconcile_racks.go

```diff
@@ -2243,6 +2246,52 @@ func (rc *ReconciliationContext) fixMissingPVC() (bool, error) {
 	return false, nil
 }
 
+func (rc *ReconciliationContext) SetFullQueryLogging() result.ReconcileResult {
```

Comment:
Is the visibility supposed to be public?

Reply:
I'm open to your thoughts on this. The other functions called within `ReconcileAllRacks()` are public, so I took my cue from there.